test/index.spec.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env mocha */
+'use strict'
+
+var expect = require('chai').expect
+var IPLD = require('../src').IPLD
+
+describe('IPLD', () => {
+  it('constructor', () => {
+    var file = new IPLD({
+      name: 'hello.txt',
+      size: 11
+    })
+
+    expect(
+      file.unmarshal()
```

Comment:
Was it marsheled though?

unmarshal should be something like:

`var file = IPLD.unmarshal(cborSerializedIPLDObj)`